bin/browserify-ftw.js

```diff
@@ -5,8 +5,11 @@ var util    =  require('util')
   , fs = require('fs')
   , path    =  require('path')
   , log     =  require('npmlog')
+  , xtend = require('xtend')
```

Comment:
fix formatting here please `=` should be aligned. I'm using Tabular in vim to do that.